src/actions/sortDirsActions.js

```diff
@@ -0,0 +1,25 @@
+export const SORT_DIRS_BY_NAME_ASC = "SORT_DIRS_BY_NAME_ASC"
+export const SORT_DIRS_BY_NAME_DESC = "SORT_DIRS_BY_NAME_DESC "
+export const SORT_DIRS_BY_DATE_ASC = "SORT_DIRS_BY_DATE_ASC"
+export const SORT_DIRS_BY_DATE_DESC = "SORT_DIRS_BY_DATE_DESC"
+
+
+export const makeSortDirsByNameAsc = action =>({
```

Comment:
what is a param action for ? if you only need the action type just use the action type